### ISSUE TITLE
Section 3.4: maturity also depends on how effective the ICT development processes are in producing accessible ICT.

### ISSUE THREAD

**Issue by jasonjgw** [w3c/maturity-model#155]
This issue relates to issue #91.
Fundamental to the maturity of any ICT development process in relation to accessibility is the extent to which it results in more rather than less accessible products.

Questions that I have in mind, but which are not addressed by the current proof points, include

* How well and how consistently do delivered products satisfy the accessibility-related requirements that were set for them (e.g., technical/policy compliance, outcomes of user research, user satisfaction)? This question cannot be answered by considering a conformance report for a given product at a single moment in time, such as a WCAG evaluation. However, if the evaluations (across time and across products) consistently show that the organization is meeting the accessibility requirements/standards it set for itself, then this would constitute evidence of maturity in the ICT development process. Of course, standards are only one aspect, which is why I've noted other criteria, such as user evaluations (subjective and objective).
* Is there a tendency for delivered products to require significant remediation of accessibility issues after release? The more this occurs, the less mature the development process is.
* Are the accessibility requirements defined appropriately for each product and updated over time to take into account new knowledge, standards and policies?

In sum, one can assess an ICT development process in part by looking at its "track record" of success.

**Comment by jasonjgw** [comment]
Also, to what extent does the organization's process address accessibility-related defects in a timely and effective manner?

**Comment by Helixopp** [comment]
Thank you, for your comment. We agree that attention should be called to establishing metrics within dimensions. We will explore how to ensure that is incorporated.

**Comment by jeffkline** [comment]
I propose adding the following new subtopic labeled "Dimension Goals and Metrics" to each dimension with a singe proof point underneath that says:
"Dimension related goals established, metrics defined, and progress tracked" and will add to all worksheet tabs with concurrence

**Comment by jeffkline** [comment]
Issue comment updated with proposed language for Assessment tool for all dimensions:

I propose adding the following new subtopic labeled "Dimension Goals and Metrics" to each dimension with a singe proof point underneath that says:
"Dimension related goals established, metrics defined, and progress tracked" and will add to all worksheet tabs with concurrence

Thoughts?


Regards,


[A picture containing text, black, clock  Description automatically generated]<http://strategicaccessibility.com/>
 ***@***.***
 5  1  2   .   4  2  6   .   9   7  7  9

From: David Fazio ***@***.***>
Date: Wednesday, August 21, 2024 at 10:58 AM
To: w3c/maturity-model ***@***.***>
Cc: Jeff Kline ***@***.***>, Assign ***@***.***>
Subject: Re: [w3c/maturity-model] Section 3.4: maturity also depends on how effective the ICT development processes are in producing accessible ICT. (Issue #155)

Assigned #155<https://github.com/w3c/maturity-model/issues/155> to @jeffkline<https://github.com/jeffkline>.

**Comment by jasonjgw** [comment]
I support the proposed change, which should also address some of my issues. However, I would recommend adding an introductory subsection that explains in general terms what good/appropriate goals and metrics are, while still providing flexibility in defining them. In particular, the goals and metrics should further the attainment of conformance to whatever standards and policy-based technical requirements the organization has adopted.

**Comment by jeffkline** [comment]
Thanks, Jason, for the support and additional input.

With regard to an introductory subsection on goals… Note that the model has been developed using the premise of not being too prescriptive (beyond the proof points themselves) as every organization has its own methods and culture. In other words, we provide proof points as to what needs to be done, but leave it to the organizations to determine how they should be accomplished.

With that said, there are many places in the model, beyond goals and metrics, where the need for examples to better understand proof points could assist organizations reach full maturity. There may be an opportunity to work on these once we get the model itself to “full maturity”😊.  Perhaps even an additional work effort?  But what I envision, as the model is used more broadly,  is that users share / publish best practices and techniques for reaching Optimize levels that can shared to help others in the maturity journey.

Also, FYI…I cover the topic of goals in Chapter 8 of my book<https://a.co/d/77Z1yAW> which includes some short term and long term examples. Certainly not exhaustive, but enough there to help stimulate some thought for organizations.



Regards,


[A picture containing text, black, clock  Description automatically generated]<http://strategicaccessibility.com/>
 ***@***.***
 5  1  2   .   4  2  6   .   9   7  7  9

From: jasonjgw ***@***.***>
Date: Friday, August 23, 2024 at 9:04 AM
To: w3c/maturity-model ***@***.***>
Cc: Jeff Kline ***@***.***>, Mention ***@***.***>
Subject: Re: [w3c/maturity-model] Section 3.4: maturity also depends on how effective the ICT development processes are in producing accessible ICT. (Issue #155)

I support the proposed change, which should also address some of my issues. However, I would recommend adding an introductory subsection that explains in general terms what good/appropriate goals and metrics are, while still providing flexibility in defining them. In particular, the goals and metrics should further the attainment of conformance to whatever standards and policy-based technical requirements the organization has adopted.

**Comment by Helixopp** [comment]
We have added the following Proof Point to every Dimension: Dimension-related goals are established, metrics are defined, and progress is tracked. We do not tell people what those metrics or goals should be, or how how to track them but require them to do so. We believe this addresses your issue. If you disagree please reopen
